code/game/objects/items/storage/boxes/clothes_boxes.dm

```diff
@@ -130,6 +130,24 @@
 	new /obj/item/clothing/under/ethereal_tunic/trailwarden(src)
 	new /obj/item/storage/backpack/saddlepack(src)
 
+/obj/item/storage/box/hero/journalist
+	name = "Assassinated by CIA - 1984." // Literally
```

Comment:
This could do with being a bit more... descriptive, as it's the only thing you get to see from the beacon

Reply:
Well leave your suggestion, although I would make something up as well tomorrow

Reply:
I would probably go with something like "Truth-seeking Journalist" or "Cold War Reporter"

Reply:
An award greater than the Pulitzer? something like that, to keep the peak box description